scripts/notify-ci-wave.mjs

```
@@ -0,0 +1,116 @@
+
+const headers = {
+  'content-type': 'application/json',
+  'x-6529-ci-timestamp': timestamp,
+  'x-6529-ci-signature': `sha256=${signature}`
+};
+
+if (CI_PIPELINES_ALERT_API_AUTH) {
+  headers['x-6529-auth'] = CI_PIPELINES_ALERT_API_AUTH;
+}
+
+const response = await fetch(CI_PIPELINES_ALERT_URL, {
```

ESLint

[error] 'fetch' is not defined.

(no-undef)